### ISSUE TITLE
PiT 24.0: hilla bom does not have hilla-react version

### ISSUE THREAD

**Issue by manolo** [vaadin/hilla#732]
### Description of the bug 

When migrating a  hilla-react app from 23.3 to 24.0, `hilla-react` dependency is not resolved

```
[ERROR]     'dependencies.dependency.version' for dev.hilla:hilla-react:jar is missing. @ line 82, column 21
```

- [ ] I am using the multi-module engine (`hillaEngine=true` feature flag)

### Minimal reproducible example

```
  curl -s -f 'https://start.vaadin.com/dl?&preset=react-tutorial&projectName=react-tutorial' -o react-tutorial.zip
  unzip react-tutorial.zip
  cd react-tutorial
  mvn -q versions:update-parent -DparentVersion=3.0.2
  mvn -B -q versions:set-property -Dproperty=java.version -DnewVersion=17
  mvn -B -q versions:set-property -Dproperty=hilla.version -DnewVersion=2.0.0.alpha9
  mvn
```

### Expected behaviour
application compiles

### Actual behaviour
compilation error

### Versions:
    - Vaadin / Hilla version: 2.0.0.alpha9
    - Java version: 17

**Comment by cromoteca** [comment]
@manolo that was added in https://github.com/vaadin/platform/pull/3492 which is not released yet.

**Comment by ZheSun88** [comment]
this has been released with platform 24.0.0.alpha10. 
close as fixed
